load-test-framework/src/main/java/com/google/pubsub/clients/common/MetricsHandler.java

```diff
@@ -178,9 +178,13 @@ private void createMetrics() {
       log.info("Metrics already exist.");
     }
   }
-
+  
   public synchronized void recordLatency(long latencyMs) {
```

Comment:
We do not use operator overloading wherever possible. Please undo this. It was done like that to also avoid from grabbing and releasing the lock as often when recording a batch.